### PR TITLE
feat(nimbus): add audience form for new nimbus ui

### LINKED ISSUE #10839: Audience Form
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4619)

### PR DESCRIPTION
**Pull request by jaredlockhart**
Because

* We are implementing all the forms in HTMX
* Time to do the audience form

This commit

* Adds the audience form to the new Nimbus UI

fixes #10839

image

### CODE REVIEW

File: experimenter/experimenter/experiments/models.py [hunk 1]
```diff
@@ -643,6 +646,10 @@ def treatment_branches(self):
             branches = branches.exclude(id=self.reference_branch.id)
         return list(branches)
 
+    @property
+    def is_desktop(self):
```
Comment (brennie) [inline experimenter/experimenter/experiments/models.py:650]: Maybe this should be `application_is_desktop` or similar

File: experimenter/experimenter/nimbus_ui_new/static/css/style.scss [hunk 2]
```diff
@@ -41,6 +41,15 @@
     calc(0.75em + 0.375rem) calc(0.75em + 0.375rem);
 }
 
+.bootstrap-select.is-invalid {
```
Comment (brennie) [inline experimenter/experimenter/nimbus_ui_new/static/css/style.scss:44]: 🙃

**Review by brennie — APPROVED** [review]

File: experimenter/experimenter/experiments/models.py [hunk 3]
```diff
@@ -643,6 +646,10 @@ def treatment_branches(self):
...
+        return self.application == self.Application.DESKTOP
```
Comment (yashikakhurana) [inline experimenter/experimenter/experiments/models.py:651]: oh nice, we have couple of places in the code where we can use this property now

**Review by yashikakhurana — APPROVED** [review]
Clean and nice changes, tested locally worked absolutely fine, thanks @jaredlockhart 🎉

File: experimenter/experimenter/experiments/models.py (hunk 1 above)
Comment (jaredlockhart) [reply experimenter/experimenter/experiments/models.py:650]: Oh, could be, usually you start predicates like this with the verb so `is_` `can_` `should_` etc, so it'd be more like `is_application_desktop` but I feel like `desktop` is only used in the context of the application so it seems unambiguous to me?

File: experimenter/experimenter/nimbus_ui_new/static/css/style.scss (hunk 2 above)
Comment (jaredlockhart) [reply experimenter/experimenter/nimbus_ui_new/static/css/style.scss:44]: I know it's a hassle, bootstrap-select doesn't seem to style itself consistently with the other bootstrap5 elements so I gotta keep overriding it.

**Pull request merged**